source/fundamentals/connection.txt

```diff
@@ -44,8 +44,8 @@ your MongoDB instance.
 
 The last part of the connection string contains connection and authentication
 options as parameters. In the example above, we set two connection options:
-``poolSize=20`` and ``w=majority``. 
-For more information on connection options, skip to the :ref:`Connection Options` section.
```

Comment:
Should connection options (above) also use a hyphen?

Reply:
This is a `ref` to a section within the page whereas the other instances are `doc` links that refer to this page.